broker/utils.go

```diff
@@ -25,10 +25,6 @@ func findDeployment(c compose.Client, name string) (*composeapi.Deployment, erro
 	return nil, fmt.Errorf("deployment: not found")
 }
 
-func JDBCURI(scheme, hostname, port, dbname, username, password string) string {
```

Comment:
We can add it back in as and when we have a need for it. I think it's not good to have code hanging around because we might need it in future.